robolectric/src/test/java/org/robolectric/shadows/ShadowApplicationTest.java

```diff
@@ -401,6 +402,44 @@ public void startActivity_whenActivityCheckingEnabled_checksPackageManagerResolv
       assertThat(shadowOf(application).getNextStartedActivity()).isNull();
     }
   }
+
+  @Test
+  public void bindServiceShouldAddServiceConnectionToListOfBoundServiceConnections() {
+    final ShadowApplication shadowApplication = Shadows.shadowOf(RuntimeEnvironment.application);
+    final ServiceConnection expectedServiceConnection = new EmptyServiceConnection();
+
+    assertEquals(0, shadowApplication.getBoundServiceConnections().size());
```

Comment:
Would you mind updating all of the assertions to use `assertThat`? I believe that's the style we're using going forward.